flint.ui/src/components/Footer/Footer.vue

```diff
@@ -0,0 +1,119 @@
+              target="_blank"
+            >
+              <i class="fab fa-linkedin text-2xl text-gray-400"></i>
+            </a>
+          </div>
+        </div>
+      </div>
+      <div class="flex justify-end items-start gap-20 text-base text-white font-normal tablet:justify-start">
+        <div class="grid gap-5">
+          <h1 class="text-xl font-medium">Site Links</h1>
+          <h1>
+            <a href="https://moja.global/about/" target="_blank" style="text-decoration: none; color: inherit">About</a>
+          </h1>
+          <h1>
+            <a href="https://moja.global/what-we-do/" target="_blank" style="text-decoration: none; color: inherit"
+              >What We Do</a
+            >
+          </h1>
+          <h1>
+            <a href="https://moja.global/flint/" target="_blank" style="text-decoration: none; color: inherit">FLINT</a>
+          </h1>
+          <h1>
+            <a href="https://moja.global/get-involved/" target="_blank" style="text-decoration: none; color: inherit"
+              >Get Involved</a
+            >
+          </h1>
+          <h1>
+            <a href="https://moja.global/resources/" target="_blank" style="text-decoration: none; color: inherit"
+              >Resources</a
+            >
+          </h1>
+          <h1>
+            <a href="https://moja.global/news/" target="_blank" style="text-decoration: none; color: inherit">News</a>
+          </h1>
+        </div>
+        <div class="grid grid-cols-1 w-1/3 gap-5">
+          <h1 class="text-xl font-medium">News</h1>
+          <h1>
+            <a
+              href="https://moja.global/moja-global-wins-azure-cloud-grant/"
+              target="_blank"
+              style="text-decoration: none; color: inherit"
+              >Moja Global Wins Azure Cloud Grant</a
+            >
+          </h1>
+          <h1>
+            <a
+              href="https://moja.global/generic-plan-in-sprints-to-implement-of-flint-based-mrv-system/"
+              target="_blank"
+              style="text-decoration: none; color: inherit"
+              >Generic Plan in Sprinrs To Implement of FLINT-Based MRV System</a
+            >
+          </h1>
+          <h1>
+            <a
+              href="https://moja.global/challenges-for-open-source-projects-when-working-with-public-international-organisations/"
+              target="_blank"
+              style="text-decoration: none; color: inherit"
+              >Challengers for Open Source Projects When Working With Public Interntional Organisations</a
+            >
+          </h1>
+          <button class="border-solid border-2 border-white py-2 text-xs w-2/3">
+            <a
+              style="text-decoration: none; color: inherit"
+              href="https://global.us13.list-manage.com/subscribe?u=d8a187fa0dfae0bafe4bc69cc&id=cbfdc2d476"
+              target="_blank"
+            >
+              Subscribe to our newsletter
+            </a>
+          </button>
```

Comment:
This can be removed. Our newsletter is not active as of now.

Reply:
https://github.com/moja-global/FLINT-UI/pull/168#issuecomment-1061604370

I have mentioned about it, let me know about the correct link.